src/js/components/interactiveDataSources/InteractiveDataSourcesPage.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { ComingSoon } from 'data-transparency-ui';
+import { scrollToY } from 'helpers/scrollToHelper';
+import { getStickyBreakPointForSidebar } from 'helpers/stickyHeaderHelper';
+
+import PageWrapper from 'components/sharedComponents/PageWrapper';
+
+import InteractiveDataSourcesSection from './InteractiveDataSourcesSection';
+import { interactiveDataSourcesPageMetaTags } from '../../helpers/metaTagHelper';
+import ExampleScroller from './scrollerSections/ExampleScroller';
+import Sidebar from '../sharedComponents/sidebar/Sidebar';
+
+require('pages/interactiveDataSources/index.scss');
+
+const scrollPositionOfSiteHeader = getStickyBreakPointForSidebar();
+
+const InteractiveDataSourcesPage = () => {
+    const [activeSection, setActiveSection] = useState('example-scroller');
+
+    const sections = [
+        {
+            name: 'example-scroller',
+            display: 'Example Scroller',
+            showSectionTitle: false,
+            component: <ExampleScroller />
+        }
+    ];
+
+    const jumpToSection = (section = '') => {
+        // check if valid section
+        const matchedSection = sections.find((obj) => obj.name === section);
+        if (!matchedSection) return; // no matching section
+
+        // scroll to correct section
+        const sectionDom = document.querySelector(`#interactive-data-sources-${matchedSection.name}`);
+        if (!sectionDom) return;
+
+        if (activeSection === 'intro-section') {
+            scrollToY(sectionDom.offsetTop - 150, 700);
+        }
+        else {
+            // scrollY set to the top of the section, subtracting the height of sticky elements + 20px of margin
+            scrollToY(sectionDom.offsetTop - 86, 700);
+        }
+
+        setActiveSection(matchedSection.name);
+    };
+
+    return (
+        <PageWrapper
+            pageName="Data Sources"
+            classNames="usa-da-interactive-data-sources-page"
+            overLine="resources"
+            metaTagProps={interactiveDataSourcesPageMetaTags}
+            title="Data Sources">
+            <main id="main-content" className="main-content usda__flex-row">
+                <div className="sidebar usda__flex-col">
+                    <Sidebar
+                        pageName="interactive-data-sources"
+                        fixedStickyBreakpoint={scrollPositionOfSiteHeader}
+                        isGoingToBeSticky
+                        active={activeSection}
+                        jumpToSection={jumpToSection}
```

Comment:
Is there a shorthand you can use in React for situations like this where the variable name matches the prop? E.g. just `jumpToSection` or `{jumpToSection` or something like that?

Reply:
Not for one variable specifically unless it's a boolean value. However, you can use a spread operator if the variables are contained in an object, such as this.props like the example below:
`<SomeElem {...this.props} />`